.travis.yml

```diff
@@ -2,4 +2,30 @@ language: java
 sudo: false
 cache:
   directories:
-      - $HOME/.m2/repository
+    - "$HOME/.m2/repository"
+
+install: mvn install -DskipTests=true -Dmaven.javadoc.skip=true
+script:
+  - jdk_switcher use oraclejdk8
+  - mvn test
+  - jdk_switcher use oraclejdk7
+  - mvn test
+before_deploy: bin/build-release.sh
+
+matrix:
+  fast_finish: true
+  include:
+    - env: STORM_RELEASE="0.10.0" MESOS_RELEASE="0.28.0"
+    - env: STORM_RELEASE="0.10.0" MESOS_RELEASE="0.27.2"
+    - env: STORM_RELEASE="0.9.6" MESOS_RELEASE="0.28.0"
+    - env: STORM_RELEASE="0.9.6" MESOS_RELEASE="0.27.2"
+
+# Deploy archives to Github release tags
+deploy:
+  provider: releases
+  api_key:
+    secure: DMwskItRMoVMpmYt3trBy4WlQN37ZnT28p1jYhP6xQOD1CfgZC+ggGiWr+aobfD89ZvztGdRPXyBCorsErqXEPLIA/dSAfJaOr20o6KzyFz284vw08dLwIvjev/eRYuT5BuRVCyFkt4MkTQEEGJdBE1Fb0nc8xdaylDR6CqthyI/qEE1dx4/yFoyBTBHQOBI3EynOG+h3D6XKQCfheNyX3i63St2wyI0hfTnanGNpsdxwgO5H4SW7p48/f+z4kjIt9IxHL7Lr8b56WbKvo97OqXzn+OFiCO8m/DtDD+HTUqjwDQA5oYvMx9bBHmZEyz/HgTnNsgH0RqTfo97AqezV37LSzJyWupF6Nk5N3k+UJu5ck/EvaiOBjDsye9sM1O6W5XICUfgNKHHvx6Ow5nVBY7gyrcCsJ+w65sda6Ij731GxmWEv2u9rfpq0/mNRvPwf/wFS9JN7SwNzsJPTYswsNyJoH7o/SVH7CiRavUTyCJJvelSimY9GwOWBCVNQIR8cioOncqH1mp1oxbSAVutcoppUt/jDNrxQwFM+XeI8AifK/CwoujSR6e3QKCPyRh3t5QICHwDcAh5qKDNGrLKEjTejCX+MdDKro+qzh4w7/T8jcZEKS+TpPC3cnFiT5N4joJ0EQ7oENRLpTR2JA4K38EPMSluma2dzveTXdz74QE=
```

Comment:
I know you mentioned that we'd need to update the api_key after merging, but I'd like a bit more info on this.  Can you please explain how this key was generated, and whether it seems safe/acceptable to have it open for view/editing by non-members of the group?  I'm guessing we don't wanna have it just sitting in our repo open to the world, since I'm assuming that would let anyone do API requests to the mesos/storm repo and potentially break it.

Perhaps we should leverage [travis's support for encrypting stuff](https://docs.travis-ci.com/user/encryption-keys/)? (Which I know *nothing* about!)